<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-U7APJPA" failures="1" tests="2" name="com.qa.testcases.dbx.autoactions.TC_DBX_AA_04" time="9.876" errors="1" timestamp="17 Jun 2022 11:36:33 GMT" skipped="0">
  <testcase classname="com.qa.testcases.dbx.autoactions.TC_DBX_AA_04" name="tearDown" time="1.832">
    <error message="java.net.UnknownHostException: No such host is known (grafanaui.unraveldata.com)" type="org.influxdb.InfluxDBIOException">
      <![CDATA[org.influxdb.InfluxDBIOException: java.net.UnknownHostException: No such host is known (grafanaui.unraveldata.com)
at org.influxdb.impl.InfluxDBImpl.parseHost(InfluxDBImpl.java:237)
at org.influxdb.impl.InfluxDBImpl.<init>(InfluxDBImpl.java:147)
at org.influxdb.impl.InfluxDBImpl.<init>(InfluxDBImpl.java:123)
at org.influxdb.impl.InfluxDBImpl.<init>(InfluxDBImpl.java:187)
at org.influxdb.InfluxDBFactory.connect(InfluxDBFactory.java:30)
at com.qa.connections.db.InfluxDBClient.<init>(InfluxDBClient.java:27)
at com.qa.connections.db.InfluxDBClient.getConnection(InfluxDBClient.java:42)
at com.qa.base.BaseClass.tearDown(BaseClass.java:157)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.UnknownHostException: No such host is known (grafanaui.unraveldata.com)
at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:929)
at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1515)
at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:848)
at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1505)
at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1364)
at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1298)
at java.base/java.net.InetAddress.getByName(InetAddress.java:1248)
at org.influxdb.impl.InfluxDBImpl.parseHost(InfluxDBImpl.java:235)
... 27 more
]]>
    </error>
  </testcase> <!-- tearDown -->
  <testcase classname="com.qa.testcases.dbx.autoactions.TC_DBX_AA_04" name="TC_DBX_AA_04_deleteExistingPolicy" time="8.044">
    <failure message="Policy not deleted expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Policy not deleted expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at com.qa.scripts.databricks.autoaction.DbxAutoAction.deleteJob(DbxAutoAction.java:122)
at com.qa.testcases.dbx.autoactions.TC_DBX_AA_04.TC_DBX_AA_04_deleteExistingPolicy(TC_DBX_AA_04.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_DBX_AA_04_deleteExistingPolicy -->
</testsuite> <!-- com.qa.testcases.dbx.autoactions.TC_DBX_AA_04 -->
